src/app/modules/features/toDoList/components/toDoList/toDoList.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="d-flex flex-column step-creation-wrapper">
+    <div>
+        <!-- toDo translate -->
+        <h2> Liste des taches</h2>
+        <table>
+            <tbody *ngFor="let task of taskList; let idx = index">
+                <p *ngIf="idx !== indexList"> {{ task.title }} </p>
+                <p>
+                    <input *ngIf="idx === indexList"
+                        nbInput
+                        [(ngModel)]="taskName" 
+                        (keyup.enter)="changeName(task.id,taskName)"
+                    >
+                </p>
+                <ng-container class="state" *ngIf="task.state === toDo">
+                    <select nbSelect>
+                        <option (click)="changeState(toDo, task.id)" selected>A faire</option>
+                        <option (click)="changeState(doing, task.id)">En cours</option>
+                        <option (click)="changeState(done, task.id)">Fait</option>
```

Comment:
Using (click) on the options do not trigger any action or function.. Need to use ngModel or smthg like that (check doc)

There is an exemple in the user-settings.component.html (language selection)